### PR TITLE
feat: coderabbit.ai config

### PR DESCRIPTION
**Pull request by justinr1234**

### CODE REVIEW

File: .coderabbit.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  review_status: true
+  # Generate walkthrough in a markdown collapsible section.
+  collapse_walkthrough: false
+  # Abort the in-progress review if the pull request is closed or merged.
+  abort_on_close: true
+  auto_review:
+    # Automatic Review | Automatic code review
+    enabled: true
+    # Review draft PRs/MRs.
+    drafts: false
```
Comment (ckeshava) [inline .coderabbit.yaml:22]: I could use code-rabbit's reviews on draft PRs. It's helpful to get a round of comments before I open up the PRs.

Is that allowed under the open-source project plan? Can we enable it?

Comment (justinr1234) [reply .coderabbit.yaml:22]: @mvadari any complaints?

Comment (mvadari) [reply .coderabbit.yaml:22]: You can manually request reviews on drafts, I'd prefer to keep it automatically off so you don't get reviews before you're ready for them.

Comment (ckeshava) [reply .coderabbit.yaml:22]: How do you manually request for PR reviews? "@coderabbit review" ?
If that's the case, I'm happy with the config 👍

**Review by pdp2121 — APPROVED** [review]

**Pull request merged**